recipes/libcufile/meta.yaml

```
@@ -0,0 +1,85 @@
+{% set name = "libcufile" %}
+{% set version = "1.5.0.59" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 222c49b29afb544c69542c4e658b0a1bd01fd2306f2136ad6990ccfff482a097  # [linux64]
+  sha256: 4e8d2b525322be9ab6ae574bd1d0325a4bf9c6c3bbdaa1611b5b969b7b34b0c4  # [aarch64]
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [ppc64le]
+  skip: true  # [aarch64]
+  skip: true  # [win]
+
+outputs:
+  - name: libcufile
+    files:
+      - lib/libcufile*.so.*
+      - etc/cufile.json
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufile*.so.*
+        - test -f $PREFIX/etc/cufile.json
+
+  - name: libcufile-dev
+    requirements:
+      run:
+        - libcufile >={{ version }}
+    files:
+      - lib/libcufile*.so
+      - targets/*/lib/libcufile*.so
+      - include/cufile*
+      - targets/*/include/cufile*
+      - man
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufile*.so
+        - test -f $PREFIX/include/cufile*
+        - test -f $PREFIX/man
+
+  - name: libcufile-static
+    requirements:
+      run:
+        - libcufile-dev >={{ version }}
+    files:
+      - lib/libcufile*_static.a
+      - targets/*/lib/libcufile*_static.a
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufile*_static.a
+
+  - name: gds-tools
+    requirements:
+      run:
+        - libcufile >={{ version }}
+    files:
+      - gds/samples
+      - gds/tools
```

Think we need to recheck these paths. Seeing [errors on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=653059&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=788 ) that indicate it can't find these

---

At least looking on CI, these appear to be...

```suggestion
    files:
      - samples
      - tools
```

---

That's said, maybe we want to move these under `gds`. @adibbley, do you have thoughts here? 🙂

---

Yeah we should move `samples` and `tools` to a `gds` directory.